ports/servoshell/desktop/headed_window.rs

```diff
@@ -100,6 +106,20 @@ impl Window {
             winit_window.set_window_icon(Some(load_icon(icon_bytes)));
         }
 
+        #[cfg(target_os = "macos")]
```

Comment:
Could we move this into a `force_srgb_color_space` helper function so this main initialization remains relatively high level? Otherwise this seems fine to me.

Reply:
I can, might be a bit messy tho.